commons-app/src/main/java/cz/cuni/mff/xrg/odcs/commons/app/conf/ConfigProperty.java

```diff
@@ -32,6 +32,10 @@
     BACKEND_LOG_KEEP("backend.log.keepDays"),
     BACKEND_DEFAULTRDF("backend.defaultRdf"),
     BACKEND_LIMIT_OF_SCHEDULED_PIPELINES("backend.scheduledPipelines.limit"),
+    BACKEND_ALIVE_LIMIT("backend.alive.limit"),
```

Comment:
Please add sample use of this configuration param to config.properties (commented)